cfgov/apache/conf.d/redirects.conf

```diff
@@ -1017,8 +964,40 @@ RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/?$
 RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/planning/creativesavingsstrategies/?$ /consumer-tools/military-financial-lifecycle/
 RedirectMatch permanent (?i)^/practitioner-resources/servicemembers/protecting/?$ /consumer-tools/military-financial-lifecycle/
 
-
 # Common misspellings
 RedirectMatch (?i)(.*)morgage(.*) $1mortgage$2
 
 Redirect /assets https://s3.amazonaws.com/files.consumerfinance.gov/a/assets
+
+# Redirects, requested by Rachael Keeler in July 2019
+# https://GHE/CFGOV/platform/issues/3484
+RedirectMatch permanent (?i)^/about-us/advisory-groups/(.*) /about-us/advisory-committees/$1
+Redirect permanent /about-us/blog/understand-your-options-tips-student-loan-borrowers-disabilities /about-us/blog/tips-student-loan-borrowers/
+Redirect permanent /about-us/doing-business-with-us/small-businesses/ /about-us/doing-business-with-us/small-minority-businesses/
+Redirect permanent /about-us/newsroom/student-debt-domino-effect/ /about-us/blog/student-debt-domino-effect/
+Redirect permanent /about-us/project-catalyst/run-a-trial-disclosure/ /about-us/innovation/trial-disclosure-program/
+RedirectMatch permanent (?i)^/askcfpb/search/(.*) /ask-cfpb/search/$1
+Redirect permanent /complaints/ /data-research/consumer-complaints/
+Redirect permanent /consumer-tools/debt-collection/if-creditor-sues-you/ /ask-cfpb/what-should-i-do-if-a-creditor-or-debt-collector-sues-me-en-334/
+Redirect permanent /consumer-tools/everyone-has-a-story/dani-struggling-repay-student-debt/ /consumer-tools/everyone-has-a-story/dani-struggling-to-repay-student-debt/
+Redirect permanent /consumer-tools/money-as-you-grow/book-club/discussion-guides/ /consumer-tools/money-as-you-grow/bookshelf/
+RedirectMatch permanent (?i)^/foia/quarterly/(.*) /foia-requests/foia-electronic-reading-room/cfpb-foia-quarterly-reports/$1
+RedirectMatch permanent (?i)^/find-a-housing-counselor/(.*) /owning-a-home/$1
```

Comment:
Not sure what happened to get that ` data-original-title=` to appear in the report of a 404 at `https://www.consumerfinance.gov/find-a-housing-counselor/ data-original-title=`, but we definitely don't want to be redirecting it to OAH.